<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>796</width>
    <height>759</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>791</width>
      <height>791</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="pagestart">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="startgame">
        <property name="text">
         <string>新的开始</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="readbutton">
        <property name="text">
         <string>读取上一次保存的数据</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="closebutton">
        <property name="text">
         <string>退出游戏</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="pagemain">
     <widget class="QProgressBar" name="daytbar">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>771</width>
        <height>16</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
      <property name="textVisible">
       <bool>false</bool>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="invertedAppearance">
       <bool>false</bool>
      </property>
      <property name="textDirection">
       <enum>QProgressBar::TopToBottom</enum>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget_2">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>470</y>
        <width>231</width>
        <height>281</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="buttonlayout">
       <item>
        <widget class="QPushButton" name="startpig">
         <property name="text">
          <string>开始养猪</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="stoppig">
         <property name="text">
          <string>暂停</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="plague_pig">
         <property name="text">
          <string>模拟猪瘟</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="geli">
         <property name="text">
          <string>加装隔离带</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="savebutton">
         <property name="text">
          <string>保存当前数据</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="sellpig">
         <property name="text">
          <string>出圈购猪</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="to_check">
         <property name="text">
          <string>查询/统计</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="back_start">
         <property name="text">
          <string>返回主页面</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QChartView" name="chartweight" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>540</y>
        <width>551</width>
        <height>201</height>
       </rect>
      </property>
     </widget>
     <widget class="QChartView" name="chartnum" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>300</y>
        <width>551</width>
        <height>241</height>
       </rect>
      </property>
     </widget>
     <widget class="QChartView" name="charfeed" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>551</width>
        <height>221</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="mainlabel_up">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>551</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="mainlabel_right">
      <property name="geometry">
       <rect>
        <x>562</x>
        <y>33</y>
        <width>221</width>
        <height>261</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="mainlabel_down">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>300</y>
        <width>221</width>
        <height>171</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="pagecheck">
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>630</y>
        <width>791</width>
        <height>131</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="1">
        <widget class="QPushButton" name="checkpig">
         <property name="text">
          <string>查询该猪</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="checkjuan">
         <property name="text">
          <string>查询猪圈</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="pig_no"/>
       </item>
       <item row="1" column="0">
        <widget class="QLineEdit" name="juanpig_no"/>
       </item>
       <item row="0" column="2">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="searchbutton">
           <property name="text">
            <string>查询近五年销售记录</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="savebutton_2">
           <property name="text">
            <string>保存当前数据</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="back_main">
         <property name="text">
          <string>返回</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="checklabel_right">
      <property name="geometry">
       <rect>
        <x>572</x>
        <y>3</y>
        <width>221</width>
        <height>621</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="pig_image">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>561</width>
        <height>621</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="pageend">
     <widget class="QLabel" name="end_image">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>11</y>
        <width>761</width>
        <height>421</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QPushButton" name="closebutton_2">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>710</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>退出</string>
      </property>
     </widget>
     <widget class="QLabel" name="endlabel_down">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>455</y>
        <width>791</width>
        <height>161</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial Black</family>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QWidget</extends>
   <header location="global">qchartview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
